generators/cypress/templates/src/test/javascript/cypress/integration/account/login-page_spec.ts.ejs

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
Every tests are `.spec.ts` instead of `_spec.ts`.
I think we should keep `.spec.ts`.

Reply:
I have tested with . instead of _ and it seems that Cypress does not recognize them

Reply:
We found out a solution for that. We have to change :
 
`"testFiles": "**/*_spec.ts" to "testFiles": "**/*.spec.ts"`

In generators/cypress/templates/cypress.json.ejs

Reply:
my bad it came from cypress configuration. let's use dot instead of dash